packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -54,7 +54,7 @@ const BlockInspector = ( {
 			<div className="editor-block-inspector__card block-editor-block-inspector__card">
 				<BlockIcon icon={ blockType.icon } showColors />
 				<div className="editor-block-inspector__card-content block-editor-block-inspector__card-content">
-					<div className="editor-block-inspector__card-title block-editor-block-inspector__card-title">{ blockType.title }</div>
+					<h2 className="editor-block-inspector__card-title block-editor-block-inspector__card-title">{ blockType.title }</h2>
 					<div className="editor-block-inspector__card-description block-editor-block-inspector__card-description">{ blockType.description }</div>
```

Comment:
Hi @nicolad thank you for your contribution 👍 
Would it make sense to use a span instead of a div for the blockType.description container?

Reply:
Hello @jorgefilipecosta have replaced `div` with `span`. Thank you for feedback.